src/App.tsx

```diff
@@ -1,10 +1,80 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Form } from './components/Form';
+import { TodoList } from './components/Todolist';
+import { Footer } from './components/Footer';
+import { getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 6910;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [title, setTitle] = useState<string>('');
+  const [completed] = useState<boolean>(false);
+  const [filter, setFilter] = useState('all');
+  const [error, setError] = useState('');
+  const [showError, setShowError] = useState(false);
```

Comment:
you don't need 2 error state variables
```suggestion
  const [error, setError] = useState('');
  const showError = !!error || error !== '';
```